modules/visual_script/visual_script_editor.cpp

```diff
@@ -1040,7 +1041,7 @@ void VisualScriptEditor::_member_button(Object *p_item, int p_column, int p_butt
 			} else if (p_button==0) {
 
 				signal_editor->edit(name);
-				edit_signal_dialog->set_title(TTR("Editing Signal:")+" "+name);
+				edit_signal_dialog->set_title(TTR("Editing Signal: " + name));
```

Comment:
Well, guess I was mistaken. I still prefer the `vformat()` though, since makes it clearer for translators that there's a signal name in the same phrase.